var/spack/repos/builtin/packages/berkeley-db/package.py

```diff
@@ -39,6 +39,7 @@ class BerkeleyDb(AutotoolsPackage):
     build_directory = "build_unix"
 
     patch("drop-docs.patch", when="~docs")
+    patch("tls.patch")
```

Comment:
Can you add a comment above the `patch` directive with provenance, aim of the patch etc. ?

Reply:
Done.

Reply:
Done.